### DISCUSSION TITLE
关于全球震动软件的运行内存问题 [About GlobalQuake RAM Issuse]

### DISCUSSION THREAD

**Discussion by CRH380A175** [xspanger3770/GlobalQuake#220] [Q&A]
**中文原文**
我下载了 GlobalQuake 软件并对其进行了本地模式运行，
但发现了一些问题： 我的电脑总RAM为16GB，如图所示：
![image](https://github.com/xspanger3770/GlobalQuake/assets/88586120/9fae082c-3126-466f-bb1f-63f78b4ce2ec)

但是，在使用软件的过程中，发现它只支持1~2GB RAM容量，这与github上GlobalQuake教程描述中“所谓软件使用了最大RAM容量的50%”的描述严重不一致， 如图所示：
![image](https://github.com/xspanger3770/GlobalQuake/assets/88586120/521c7b17-4b8a-4bea-b760-9f283fca566f)
这是怎么回事？我该如何解决这个问题？

我正在使用的GlobalQuake软件版本如图所示：
![image](https://github.com/xspanger3770/GlobalQuake/assets/88586120/cfea4d6c-a45d-435b-a950-7468fae62751)

我翻遍了整个互联网，就是找不到相关问题的解决方案。
我希望有一个教程来解决这个问题！我将感激不尽。

**English**
I downloaded the GlobalQuake software and ran it with localization, 
but I found some issues:
My computer has a total RAM of 16GB, as shown in Fig:
![Weixin Image_20240130210758](https://github.com/xspanger3770/GlobalQuake/assets/88586120/7233104d-2c44-4dcb-960c-e78e04b1a681)
However, in the process of using the software, it was found that it only supports 1~2GB RAM capacity, which is seriously inconsistent with the description of "the so-called software uses 50% of the maximum RAM capacity" in the tutorial description of GlobalQuake on github, as shown in Fig:
![image](https://github.com/xspanger3770/GlobalQuake/assets/88586120/9294c7dd-c86d-4851-817e-e10e825ffd6d)
What's going on? How do I fix this?
The version of GlobalQuake software I am using is the one shown in the image:
![image](https://github.com/xspanger3770/GlobalQuake/assets/88586120/99131ac2-23f1-454c-8514-7a451ba41953)
I searched the whole internet and just couldn't find a solution to the problem.
I would appreciate a tutorial to solve the problem!
The English translation of the above Chinese question is from: 
Microsoft Bing Translate. FYI.

**Comment by YacineBoussoufa** [comment]
In version 0.10.0 GlobalQuake used around (min-max) 50-70% of RAM, a lot of users noted that having only 8GB RAM, their PC started to lag with GlobalQuake open. In the installer you have highlited the RAM usage was decreased to 50-60%. Users again complained about that GlobalQuake was so slow and used so few RAM. 
We decided to release a Version 2 of the installer (you can see it on top of the one you highlighted) this one has a 65-70 limit. You could try this last one and see if it got resolved. Link here: https://github.com/xspanger3770/GlobalQuake/releases/download/v0.10.1/GlobalQuake_v0.10.1_Installer_v2_Windows.exe

Or you could download the GlobalQuake_v0.10.1.jar file and run it via command prompt or bat file with
`java -Xmx10G -jar GlobalQuake_v0.10.1.jar` 
where "-Xmx10G" means that the maximum RAM limit is 10Gb.

------------------ Chinese translated with DeepL -- 中文翻译，附 DeepL ------------------
在 0.10.0 版本中，GlobalQuake 占用了大约（最小-最大）50%-70% 的内存，很多用户指出，如果只有 8GB 内存，他们的电脑在打开 GlobalQuake 时就会开始卡顿。在安装程序中，内存使用率已降至 50-60%。用户再次抱怨 GlobalQuake 运行速度太慢，内存占用太少。
我们决定发布安装程序的第 2 版（您可以在您突出显示的安装程序上看到它），该版本的内存限制为 65-70。你可以试试最后一个版本，看看问题是否得到了解决。链接地址： https://github.com/xspanger3770/GlobalQuake/releases/download/v0.10.1/GlobalQuake_v0.10.1_Installer_v2_Windows.exe

或者您可以下载 GlobalQuake_v0.10.1.jar 文件，并通过命令提示符或 bat 文件运行它，方法是
`java -Xmx10G -jar GlobalQuake_v0.10.1.jar`. 
其中"-Xmx10G "表示最大 RAM 限制为 10Gb。
